frontend/src/component/layout/MainLayout/MainLayout.tsx

```diff
@@ -74,9 +78,33 @@ const MainLayoutContentContainer = styled('div')(({ theme }) => ({
     zIndex: 200,
 }));
 
+const SegmentsSplash: React.FC = () => {
+    const { value: localStorageState, setValue: setLocalStorageState } =
+        createLocalStorage('OssSegmentsSplashScreen:v1', { shown: false });
+
+    const [showSegmentSplash, setShowSegmentSplash] = React.useState(true);
+
+    const navigate = useNavigate();
+    const closeSegmentsSplash = () => {
+        setShowSegmentSplash(false);
+        setLocalStorageState({ shown: true });
+    };
+
+    return (
+        <SegmentsSplashScreen
+            open={showSegmentSplash && !localStorageState.shown}
+            onClose={closeSegmentsSplash}
+            showSegments={() => {
+                closeSegmentsSplash();
+                navigate(`/segments`);
+            }}
```

Comment:
Do we want to add a Plausible event here?

Reply:
Ah, that's a good point. Yeah, we could definitely do that. What do you think we should track? Just the how many people see it?

Reply:
Yeah, tracking an event here would let us see how many people click it, but it depends on whether we consider it's worth tracking that 🤷 

So, something similar to the `trackEvent`s you'll find in `Demo.tsx` for example.